gulpfile.ts

```diff
@@ -34,6 +34,28 @@ gulp.task("build", () => {
   );
 });
 
+gulp.task("bnc", () => {
+  runSequence(
+    "clean",
+    "compile",
+    "place-fixture",
+    "browserify",
+    "uglify",
+    "coverage"
+  );
+});
+
+gulp.task("bnt", () => {
```

Comment:
I'll add a comment sounds good - I'd prefer to keep it short so I don't have to type too much :P

---
In reply to: [119240679](https://github.com/Microsoft/clarity-js/pull/9#discussion_r119240679) [](ancestors = 119240679)